IdentityCore/src/webview/embeddedWebview/MSIDOAuth2EmbeddedWebviewController.m

```diff
@@ -377,6 +377,22 @@ - (void)decidePolicyForNavigationAction:(WKNavigationAction *)navigationAction
         return;
     }
     
+    // Handle anchor links that were clicked
+    if([navigationAction navigationType] == WKNavigationTypeLinkActivated) {
+#if TARGET_OS_IPHONE
+        UIApplication *application = [UIApplication performSelector:@selector(sharedApplication)];
```

Comment:
Can we use MSIDAppExtensionUtil?

Reply:
Agree that openURL logic should be present in a Utility class if it is going to be used at many places. I've modified MSIDAppExtensionUtil to include logic for macOS target.